sql/core/src/test/scala/org/apache/spark/sql/MetadataCacheSuite.scala

```diff
@@ -41,7 +41,7 @@ abstract class MetadataCacheSuite extends QueryTest with SharedSQLContext {
   test("SPARK-16336,SPARK-27504 Suggest doing table refresh " +
     "when encountering FileNotFoundException") {
```

Comment:
The test case name `Suggest doing table refresh` is not valid anymore DSv2 after this PR.